client_end_user/shared/build.gradle.kts

```diff
@@ -72,6 +72,8 @@ kotlin {
                 api(libs.androidx.core.ktx)
                 api(libs.koin.android)
                 implementation(libs.androidx.constraint)
+                implementation(libs.google.accompanist)
```

Comment:
as i know accompanist libirary became deprecated or we don't need it to handle status bar, "edge to edge" is the replacement of it, check this:
https://developer.android.com/develop/ui/views/layout/edge-to-edge

Reply:
https://developer.android.com/jetpack/compose/layouts/insets

Reply:
@AzizaHelmy I'll try now to change it